### PR TITLE
Whats New-Lumas latest#23 PR request 

### PR DESCRIPTION
**Pull request by iclaudiu98**
Tests automation: https://github.com/adrianmaciuc/break-it-playwright-javascript-project/issues/23

### CODE REVIEW

File: tests/ClaudiuIacob.spec.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+test('Verify if only four lumas items are listed on whats new page', async ({ page }) => {
+  await page.goto('https://magento.softwaretestingboard.com/what-is-new.html');
+  await expect(page).toHaveURL(/what-is-new/);
+  const productImageElements = await page.$$('.product-image-photo');
+  expect(productImageElements.length).toBe(4);
+});
+
+test('Verify if I am on the what is new page and I click on the Stellar solar jacket from latest lumas I am redirect to page../stellar-solar-jacket.html.', async ({ page }) => {
+  await page.goto('https://magento.softwaretestingboard.com/what-is-new.html');
+  await expect(page).toHaveURL(/what-is-new/);
```
Comment (adrianmaciuc) [inline tests/ClaudiuIacob.spec.js:12]: What is the purpose of this assertion?

Comment (iclaudiu98) [reply tests/ClaudiuIacob.spec.js:12]: Using this verification I verify if the actual page displayed have in url /what-is-new.

File: tests/ClaudiuIacob.spec.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { test, expect } = require('@playwright/test');
+
+test('Verify if only four lumas items are listed on whats new page', async ({ page }) => {
+  await page.goto('https://magento.softwaretestingboard.com/what-is-new.html');
+  await expect(page).toHaveURL(/what-is-new/);
+  const productImageElements = await page.$$('.product-image-photo');
+  expect(productImageElements.length).toBe(4);
+});
+
+test('Verify if I am on the what is new page and I click on the Stellar solar jacket from latest lumas I am redirect to page../stellar-solar-jacket.html.', async ({ page }) => {
```
[the remark this answers is not in the document]
Comment (iclaudiu98) [reply tests/ClaudiuIacob.spec.js]: updated

File: tests/ClaudiuIacob.spec.js (hunk 1 above)
Comment (adrianmaciuc) [reply tests/ClaudiuIacob.spec.js:12]: No need to do that.

Comment (adrianmaciuc) [reply tests/ClaudiuIacob.spec.js:12]: Pls remove it. And also for line 5 the same. No need

Comment (iclaudiu98) [reply tests/ClaudiuIacob.spec.js]: updated.

**Review by adrianmaciuc — APPROVED** [review]

**Review by adrianmaciuc — APPROVED** [review]

**Pull request merged**